common/common.cpp

```diff
@@ -1931,6 +1981,15 @@ void gpt_params_print_usage(int /*argc*/, char ** argv, const gpt_params & param
     options.push_back({ "logging",     "       --log-append",           "Don't truncate the old log file." });
 #endif // LOG_DISABLE_LOGS
 
+    options.push_back({ "control-vector" });
+    options.push_back({ "cvector", "-o,  --output FNAME",     "output file (default: '%s')", params.cvector_outfile.c_str() });
+    options.push_back({ "cvector", "--positive-file FNAME",   "positive prompts file, one prompt per line (default: '%s')", params.cvector_positive_file.c_str() });
+    options.push_back({ "cvector", "--negative-file FNAME",   "negative prompts file, one prompt per line (default: '%s')", params.cvector_negative_file.c_str() });
+    options.push_back({ "cvector", "--completions-file FNAME","completions file (default: '%s')", params.cvector_completions_file.c_str() });
+    options.push_back({ "cvector", "--completions N",         "number of lines of completions file to use (default: %d)", params.n_completions });
+    options.push_back({ "cvector", "--batch-pca N",           "batch size used for PCA. Larger batch runs faster, but uses more memory (default: %d)", params.n_pca_batch });
+    options.push_back({ "cvector", "--iter-pca N",            "number of iterations used for PCA (default: %d)", params.n_pca_iterations });
+
```

Comment:
The whitespace padding should be kept so that the arguments are vertically aligned when the help is printed:

```suggestion
    options.push_back({ "control-vector" });
    options.push_back({ "cvector",     "-o,    --output FNAME",         "output file (default: '%s')", params.cvector_outfile.c_str() });
    options.push_back({ "cvector",     "       --positive-file FNAME",  "positive prompts file, one prompt per line (default: '%s')", params.cvector_positive_file.c_str() });
    options.push_back({ "cvector",     "       --negative-file FNAME",  "negative prompts file, one prompt per line (default: '%s')", params.cvector_negative_file.c_str() });
    options.push_back({ "cvector",     "       --completions-file FNAME",
                                                                        "completions file (default: '%s')", params.cvector_completions_file.c_str() });
    options.push_back({ "cvector",     "       --completions N",        "number of lines from the completions file to use (default: %d)", params.n_completions });
    options.push_back({ "cvector",     "       --batch-pca N",          "batch size used for PCA. Larger batch runs faster, but uses more memory (default: %d)", params.n_pca_batch });
    options.push_back({ "cvector",     "       --iter-pca N",           "number of iterations used for PCA (default: %d)", params.n_pca_iterations });

```

Reply:
FYI, I also changed the example name + binary name to `llama-cvector-generator`